android-app/app/src/main/java/net/trustly/inappbrowserandroid/webchromeclient/TrustlyWebView.kt

```diff
@@ -49,6 +49,9 @@ class TrustlyWebView : LinearLayout {
                     val url = request.url.toString()
                     if (url.contains(TrustlyConstants.OAUTH_LOGIN_PATH))
                         launchUrl(view.context, url)
+
+                    // This return depends of your context, you can see the Android documentation to know more about it
```

Comment:
"This return depends on your context, visit the Android documentation to learn more about it.